.github/CODEOWNERS

```diff
@@ -112,6 +112,9 @@ gradlew.bat                                         @hashgraph/devops-ci @hashgr
 **/gradle.*                                         @hashgraph/devops-ci @hashgraph/devops-ci-committers
 **/*.gradle.*                                       @hashgraph/devops-ci @hashgraph/devops-ci-committers
 
+# JAVA module-info files
+**/*.module-info.java                               @hashgraph/platform-base @hashgraph/hedera-services @hashgraph/platform-hashgraph
```

Comment:
I think it would only match files ending with `.module-info.java` although that doesn't seem to make sense.
@rbarkerSL, can you weigh in on this?
If this line is supposed to act as a "catch-all" for the module-info.java files except the more specific ones that are explicitly set, it should be placed in the beginning of the file rather than at the end

Reply:
Oops my mistake; yes the line in the lower portion should either be removed.